modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -2,17 +2,17 @@
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 [id='virt-importing-vm-prerequisites_{context}']
-= Prerequisites for importing a virtual machine into {VirtProductName}
+= Prerequisites for importing a virtual machine
 
 ifdef::virt-importing-rhv-vm[]
-Importing a Red Hat Virtualization virtual machine into {VirtProductName} has the following prerequisites:
+Importing a virtual machine from Red Hat Virtualization (RHV) into {VirtProductName} has the following prerequisites:
 
-* If the operating system of the virtual machine does not map to a common {VirtProductName} template, you must create a custom ConfigMap to map the operating system to a common {VirtProductName} template.
-
-* If you are importing a VM with a Windows operating system, you must create a Windows {VirtProductName} template and a custom ConfigMap.
-
-* The default storage class must be NFS.
+ifeval::["{VirtVersion}" == "2.4"]
+* If you are importing a Linux VM that does not have a corresponding {product-title} template, you must create a custom ConfigMap.
+* If you are importing a Windows VM, you must create an {product-title} Windows template and a custom ConfigMap.
+endif::[]
 
+* You must check the {product-title} default storage class to ensure that it is NFS.
```

Comment:
We do not need necessarily to have the NFS storage set to default.
Need to be aware that for the RHV>CNV import only NFS is supported for CNV-2.4, 
and to be aware that when running VM import without picking a storage it will use the default storage class, if there is such.

Reply:
I understood that only VMware VM import will use default storage class if no storage class is selected.

I can remove 'default' here, so user must select NFS.